### PR TITLE
Add nf-test to glimpse_sample

### PR DESCRIPTION
**Pull request by LouisLeNezet**
## PR checklist

Nf-test for glimpse_sample

Closes #XXX 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

File: modules/nf-core/glimpse/sample/tests/main.nf.test [hunk 1]
```diff
@@ -0,0 +1,63 @@
+nextflow_process {
+
+    name "Test Process GLIMPSE_SAMPLE"
+    script "modules/nf-core/glimpse/sample/main.nf"
+    process "GLIMPSE_SAMPLE"
+    tag "glimpse"
+    tag "glimpse/sample"
+    tag "modules_nfcore"
```
Comment (sateeshperi) [inline modules/nf-core/glimpse/sample/tests/main.nf.test:8]:
```suggestion
    tag "modules"
    tag "modules_nfcore"
```

File: modules/nf-core/glimpse/sample/tests/main.nf.test [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+    options "--sample"
+
+    test("test_glimpse_sample") {
+        config "modules/nf-core/glimpse/sample/tests/nextflow.config"
```
Comment (sateeshperi) [inline modules/nf-core/glimpse/sample/tests/main.nf.test:13]: provide relative path for config here

File: modules/nf-core/glimpse/sample/tests/main.nf.test [hunk 3]
```diff
@@ -0,0 +1,63 @@
+nextflow_process {
+
+    name "Test Process GLIMPSE_SAMPLE"
+    script "modules/nf-core/glimpse/sample/main.nf"
+    process "GLIMPSE_SAMPLE"
+    tag "glimpse"
+    tag "glimpse/sample"
+    tag "modules_nfcore"
+
+    options "--sample"
```
Comment (sateeshperi) [inline modules/nf-core/glimpse/sample/tests/main.nf.test:10]: are you providing`--sample` here or in the nextflow.config ?

**Review by sateeshperi — CHANGES_REQUESTED** [review]
also make sure the tags have been removed from tests/config/pytest_modules.yml so that pytests will be skipped

File: modules/nf-core/glimpse/sample/tests/main.nf.test (hunk 3 above)
Comment (LouisLeNezet) [reply modules/nf-core/glimpse/sample/tests/main.nf.test:11]: From the nextflow config sorry.
I tried this but it doesn't work (i forgot to delete it).

File: modules/nf-core/glimpse/sample/tests/main.nf.test [hunk 4]
```diff
@@ -0,0 +1,62 @@
+nextflow_process {
+
+    name "Test Process GLIMPSE_SAMPLE"
+    script "modules/nf-core/glimpse/sample/main.nf"
+    process "GLIMPSE_SAMPLE"
+    tag "glimpse"
+    tag "glimpse_sample"
```
Comment (phue) [inline modules/nf-core/glimpse/sample/tests/main.nf.test:7]:
```suggestion
    tag "glimpse/sample"
```

Minor nitpick, otherwise looks good

Comment (LouisLeNezet) [reply modules/nf-core/glimpse/sample/tests/main.nf.test:7]: If I understood well this tag need to be the same as the one used in tags.yml
And in the other modules (and the one already merged) we used the name of the tool.

Comment (LouisLeNezet) [reply modules/nf-core/glimpse/sample/tests/main.nf.test:7]: I discussed with @sateeshperi  and I will update all the PR to `tool/subtool`

**Review by Emiller88 — APPROVED** [review]
Awesome, thanks again!

**Pull request merged**